classes/class-pmpro-subscription.php

```diff
@@ -1041,6 +1045,15 @@ public function set( $property, $value = null ) {
 		$this->{$property} = $value;
 	}
 
+	/**
+	 * Get the membership Level ID for this subscription.
+	 *
+	 * @since 3.1
+	 */
+	public function get_membership_level_id() {
+		return $this->membership_level_id;
```

Comment:
We need the Level ID in the callback function. It was either adding this getter, or changing the property's access to `public` instead of `protected`.